libopenage/build.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+#include "build.h"
+
+#include <libopenage/config.h>
```

Comment:
> we could add the libopenage/ to -I, to then include "config.h", though.
This was my initial attempt, but libopenage/config.h looked more explicit and easy to understand.